src/neha_Rathi/BankingSystem.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Assignment9
+ */
+package neha_Rathi;
+
+public class BankingSystem {
+	String name;
+	int amount;
+	int debitCount = 0;
+	int creditCount = 0;
```

Comment:
not require to initialise the instance variables, if not initialised variable then it will take default value of it.